tests/utils/test_comparesync.py

```diff
@@ -0,0 +1,121 @@
+# Copyright 2015 Cisco Systems, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from nose.tools import with_setup
+from ..connection.info import custom_setup, custom_teardown
+
+from ucsmsdk.utils.comparesyncmo import compare_ucs_mo, write_mo_diff
+
+
+handle = None
+ref_handle = None
+diff_handle = None
+
+
+def static_setup():
+    from ucsmsdk.ucshandle import UcsHandle
+    from ucsmsdk.ucscoremeta import UcsVersion
+
+    global ref_handle
+    global diff_handle
+
+    ref_ip = "192.168.1.1"

```

Comment:
@ragupta-git minor. these need not be defined separately, the IPs can be directly specified in the UcsHandle instantiation in `UcsHandle(ref_ip, "admin", "password")`